### PR TITLE
set required deal.II version to 9.2.0, update testers

### PR DESCRIPTION
**Pull request by tjhei**
- docker image in contrib/ci/ changed to 9.2.0
- require 9.2.0 in CMakeLists

part of #3582

### CODE REVIEW

**Comment by tjhei** [comment]
This will require inspecting and updating test output and we should wait with merging for a bit.

File: Jenkinsfile.cig [hunk 1]
```diff
@@ -102,9 +102,10 @@ pipeline {
         -G 'Ninja' \
         -D CMAKE_CXX_FLAGS='-Werror --coverage' \
         -D ASPECT_TEST_GENERATOR='Ninja' \
-        -D ASPECT_USE_PETSC='OFF' \
+        -D ASPECT_PRECOMPILE_HEADERS=OFF \
+        -D CMAKE_UNITY_BUILD=OFF \
+	-D ASPECT_USE_PETSC='OFF' \
```
Comment (gassmoeller) [inline Jenkinsfile.cig:107]:
```suggestion
        -D ASPECT_USE_PETSC='OFF' \
```

**Comment by tjhei** [comment]
some tests seem to be hanging, periodic_box2.prm for example. I am not quite sure what is going on and I don't have time to debug things right now. I will get to it in a couple of days.

**Comment by tjhei** [comment]
might be related to #3595

**Comment by tjhei** [comment]
oops, also hangs with deal.II 9.2.0 in the VM!

**Comment by tjhei** [comment]
blocked by #3604

**Comment by gassmoeller** [comment]
#3604 is closed, can this move forward?

**Comment by tjhei** [comment]
updated. Let's see if it works.

**Comment by tjhei** [comment]
> can this move forward?

I think we should merge soon so we can work on things during the workshop. Agreed?

**Comment by gassmoeller** [comment]
Agreed. I just introduced another conflict by merging #3611, can you rebase again? Sorry for that.

**Pull request merged**